types/chance/index.d.ts

```diff
@@ -92,6 +92,7 @@ declare namespace Chance {
         tld(): string;
         twitter(): string;
         url(opts?: Partial<UrlOptions>): string;
+        mac_address(opts?: Partial<MacOptions>): string;
```

Comment:
```suggestion
        mac_address(opts?: MacOptions): string;
```
I think this was wrong decision to type input params as terse interfaces.
Can we switch to optional props on this interface? thx!

Reply:
the usage of `Partial` scales better. w/o it, you would have to mark _every_ prop in most of these interfaces as optional.